2015/03/31 16:11:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/admin".' in D:\tools\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#1 D:\tools\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\project\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=user/admin
---
2015/03/31 16:11:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in D:\tools\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 D:\tools\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\project\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=user
---
2015/03/31 16:47:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in D:\tools\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 D:\tools\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\project\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=user
---
2015/03/31 16:47:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/admin".' in D:\tools\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#1 D:\tools\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\xampp\project\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=user/admin
---
2015/04/03 10:21:06 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "D:\tools\xampp\htdocs\HResource\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CAssetManager.php:138
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('D:\\tools\\xampp\\...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 D:\tools\xampp\htdocs\HResource\lib\yii\framework\gii\GiiModule.php(143): CAssetManager->publish('D:\\tools\\xampp\\...')
#3 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CComponent.php(111): GiiModule->getAssetsUrl()
#4 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CModule.php(105): CComponent->__get('assetsUrl')
#5 D:\tools\xampp\htdocs\HResource\lib\yii\framework\gii\views\layouts\main.php(5): CModule->__get('assetsUrl')
#6 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CBaseController.php(126): require('D:\\tools\\xampp\\...')
#7 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\tools\\xampp\\...', Array, true)
#8 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('D:\\tools\\xampp\\...', Array, true)
#9 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CBaseController.php(214): COutputProcessor->run()
#12 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#13 D:\tools\xampp\htdocs\HResource\lib\yii\framework\gii\views\layouts\column1.php(7): CBaseController->endContent()
#14 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CBaseController.php(126): require('D:\\tools\\xampp\\...')
#15 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\tools\\xampp\\...', Array, true)
#16 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CController.php(784): CBaseController->renderFile('D:\\tools\\xampp\\...', Array, true)
#17 D:\tools\xampp\htdocs\HResource\lib\yii\framework\gii\controllers\DefaultController.php(47): CController->render('login', Array)
#18 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionLogin()
#19 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#21 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(282): CController->run('login')
#23 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#25 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/HResource/index.php?r=gii/default/login
---
2015/04/03 10:47:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Inventory/ListItem".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Inventory/ListI...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Inventory/ListItem
HTTP_REFERER=http://localhost:9999/HResource/
---
2015/04/03 10:49:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Inventory/ListItem".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Inventory/ListI...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Inventory/ListItem
HTTP_REFERER=http://localhost:9999/HResource/index.php?r=Inventory/ListItem
---
2015/04/03 10:51:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Inventory/ListItem".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Inventory/ListI...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Inventory/ListItem
HTTP_REFERER=http://localhost:9999/HResource/index.php?r=Inventory/ListItem
---
2015/04/03 10:52:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Inventory/ListItem".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Inventory/ListI...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Inventory/ListItem
HTTP_REFERER=http://localhost:9999/HResource/index.php?r=Inventory/ListItem
---
2015/04/03 10:54:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Store/ListProduct".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Store/ListProdu...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Store/ListProduct
HTTP_REFERER=http://localhost:9999/HResource/index.php?r=employee/Show
---
2015/04/03 10:54:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Store/ListProduct".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Store/ListProdu...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Store/ListProduct
HTTP_REFERER=http://localhost:9999/HResource/index.php?r=employee/Show
---
2015/04/03 11:03:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Store/ListProduct".' in D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\tools\xampp\htdocs\HResource\lib\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Store/ListProdu...')
#1 D:\tools\xampp\htdocs\HResource\lib\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 D:\tools\xampp\htdocs\HResource\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/HResource/index.php?r=Store/ListProduct
HTTP_REFERER=http://localhost:9999/HResource/index.php?r=employee/Show
---
